forge-gui/res/cardsfolder/f/foxfire.txt

```diff
@@ -2,7 +2,9 @@ Name:Foxfire
 ManaCost:2 G
 Types:Instant
 A:SP$ Untap | Cost$ 2 G | ValidTgts$ Creature.attacking | TgtPrompt$ Select target attacking creature | SubAbility$ DBPump | SpellDescription$ Untap target attacking creature. Prevent all combat damage that would be dealt to and dealt by that creature this turn. Draw a card at the beginning of the next turn's upkeep.
-SVar:DBPump:DB$ Pump | Defined$ Targeted | KW$ Prevent all combat damage that would be dealt to and dealt by CARDNAME. | SubAbility$ DelTrigSlowtrip
+SVar:DBPump:DB$ Effect | ReplacementEffects$ RPrevent1,RPrevent2 | RememberObjects$ Targeted | ExileOnMoved$ Battlefield | SubAbility$ DelTrigSlowtrip
```

Comment:
Probably `SpellDescription$ Prevent all combat damage that would be dealt to and dealt by that creature this turn.` and cut that and following from line 4